vision/spring-framework/src/main/resources/static/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Google Cloud Vision API with Spring code sample</title>
+</head>
+<body>
+
+<div>
+    <h1>Image Label Annotations</h1>
+    <p>Returns labels classifying the content of the image:</p>
+    <form action="/extractLabels">
+        Web URL of image to analyze: <input type="text" name="imageUrl"/> <input type="submit"/>
```

Comment:
Can you refill with some image. Maybe from wikipedia?

Reply:
Done.